test/scraper/brandenburg_scraper_overview_test.rb

```diff
@@ -3,57 +3,100 @@
-    item = @scraper.extract_items(body).first
-    meta_block = @scraper.extract_meta_block(item)
-    meta_rows = @scraper.extract_meta_rows(meta_block)
-    published_at = @scraper.extract_published_at(meta_rows.last)
-    assert_equal Date.parse('2015-03-19'), published_at
+    body = @scraper.extract_body(@detail)
+    item = @scraper.extract_detail_item(body)
+    meta_row = @scraper.extract_meta_row(item)
+    published_at = @scraper.extract_published_at(meta_row.text)
+    assert_equal Date.parse('2015-02-13'), published_at
   end
 
-  test 'extract complete paper' do
-    body = @scraper.extract_body(@html)
-    item = @scraper.extract_items(body).last
-    paper = @scraper.extract_paper(item)
+  test 'extract overview paper' do
+    body = @scraper.extract_body(@overview)
+    item = @scraper.extract_overview_items(body).last
+    paper = @scraper.extract_paper_overview(item)
 
     assert_equal(
       {
         legislative_term: '6',
-        full_reference: '6/885',
-        reference: '885',
+        full_reference: '6/39',
+        reference: '39',
+        is_answer: true
+      }, paper)
+    end
+
+  test 'extract detail paper' do
+    body = @scraper.extract_body(@detail)
+    item = @scraper.extract_detail_item(body)
+    paper = @scraper.extract_detail_paper(item)
+
+    assert_equal(
+      {
+        legislative_term: '6',
+        full_reference: '6/420',
+        reference: '420',
         doctype: Paper::DOCTYPE_MINOR_INTERPELLATION,
-        title: 'Sanierung von Ortsdurchfahrten entlang der Landesstraßen 154 und 155',
-        url: 'http://www.parldok.brandenburg.de/parladoku/w6/drs/ab_0800/885.pdf',
-        published_at: Date.parse('2015-03-16'),
-        originators: { people: ['Dr. Jan Redmann', 'Rainer Genilke'], parties: ['CDU'] },
+        title: 'Umzug der Schulaufsicht von Perleberg nach Neuruppin',
+        url: 'http://www.parldok.brandenburg.de/parladoku/w6/drs/ab_0400/420.pdf',
+        published_at: Date.parse('2015-02-13'),
+        originators: {
+          people: ['Gordon Hoffmann'],
+          parties: ['CDU']
+        },
         is_answer: true
       }, paper)
   end
+
+  test 'multiple parties on major interpellations' do
+
+    meta = <<-END.gsub(/^ {6}/, '').sub(/\n$/, '')
+                  GrAnfr 10   (SPD,DIE LINKE)  13.01.2015 Drs
+                  6/420 (1 S.)Antw   (LReg)  13.02.2015 Drs
+                  6/618 (3 S.)
+    END
+    assert_equal({ people: [], parties: ['SPD', 'DIE LINKE'] }, @scraper.extract_originators(meta,
+                                                                                             Paper::DOCTYPE_MAJOR_INTERPELLATION))
+  end
+
+  test 'multiple people on minor interpellation' do
+    meta = <<-END.gsub(/^ {6}/, '').sub(/\n$/, '')
+              KlAnfr 179   Gordon Freeman (CDU), Black Widow (DIE LINKE)  13.01.2015 Drs
+              6/420 (1 S.)Antw   (LReg)  13.02.2015 Drs
+              6/618 (3 S.)
+    END
+    assert_equal({
+      people:
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.